hoot-core/src/main/cpp/hoot/core/elements/ElementData.cpp

```diff
@@ -36,5 +38,7 @@ quint64 ElementData::TIMESTAMP_EMPTY = 0;
 QString ElementData::USER_EMPTY = QString();
 long ElementData::UID_EMPTY = 0;
 bool ElementData::VISIBLE_EMPTY = true;
+double ElementData::CIRCULAR_ERROR_EMPTY = -1;
+double ElementData::CIRCULAR_ERROR_DEFAULT = ConfigOptions().getCircularErrorDefaultValue();
```

Comment:
Here is the big change getting the `CIRCULAR_ERROR_DEFAULT` from the configuration options.